docker/docker-compose.yml

```diff
@@ -133,8 +133,9 @@ services:
       - vcr-wallet:/home/indy/.indy_client/wallet
     networks:
       - vcr
+    # when running "./manage scale" comment out or remove these port mappings:
     ports:
-      - "${API_HTTP_PORT}:8080"
+      - "${EXPOSED_API_HTTP_PORT}:8080"
```

Comment:
When instances are scaled up, is it not possible to view the UI in this case?

Reply:
Yes, you can get to the UI through caddy, which is exposed on 8080

http://localhost:8080 gets you the UI